chapters/hub/hub-api.md

```diff
@@ -0,0 +1,67 @@
+# Hub API
+
+Jina Hub exposes several handy commands, namely `list`, `build` and `push` with configurable parameters. These APIs constitute the user’s experience using `jina cli` for a seamless experience with the Hub.
+
+
+## `jina hub` installation:
+`hub cli` can be installed using pip, run `pip install "jina[hub]"`
```

Comment:
Could replace this whole sentence with

```sh
pip install jina[hub]
```

It doesn't really need additional context